tests/trading/test_wcl.py

```diff
@@ -10,8 +10,9 @@
 YES = 1
 NO = 0
 
-BID = 1
-ASK = 2
+BID = 0
+ASK = 1
```

Comment:
Any particular reason these aren't pulling from `constants` like the other files?

Reply:
nope, will fix!